CHANGELOG.md

```diff
@@ -68,17 +68,29 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 - Middleware support (for e.g. for centrally handling authorization).
 
 ## [1.1.0] - 2019-05-30
+### Added
+- Travis builds for tags.
 
 ## [1.0.6] - 2019-05-19
+### Added
+- Custom logger per route support.
+
+### Fixed
+- Fixed buffer reading when timeout or disconnect occurs.
+- Fixed cookie formatting: stop url-encoding cookie path and quoting cookie expire date.
 
 ## [1.0.5] - 2018-09-03
+### Changed
+- Protocol upgrade: detaching mechanism is re-worked.
 
 ## [1.0.4] - 2018-08-31
+### Added
+- Detach callback support for protocol upgrade implementations.
 
 ## [1.0.3] - 2018-06-29
-### Added
-- Fixed eof detection
+### Fixed
+- Fixed eof detection.
 
 ## [1.0.2] - 2017-12-20
-### Added
-- Fixed request parsing with headers longer than 4096 bytes
+### Fixed
+- Fixed request parsing with headers longer than 4096 bytes.
```

Comment:
There were also: `1.0`, `1.0.1`, `0.0.1`:
Changes in 1.0: https://github.com/tarantool/http/compare/0.0.1...1.0
Changes in 1.0.1: https://github.com/tarantool/http/compare/1.0...1.0.1

I don't know is it worth to add them to changelog too or not.